examples/checkout/search/search_abandoned_transactions.rb

```diff
@@ -8,7 +8,10 @@
 #   You can pass the credentials parameters to PagSeguro::Transaction#find_abandoned
 
 options = {
-  credentials: PagSeguro::AccountCredentials.new('EMAIL', 'TOKEN')
+  credentials: PagSeguro::AccountCredentials.new('EMAIL', 'TOKEN'),
+  starts_at: (Time.now - 86400),
```

Comment:
@adbatista the code is there => https://github.com/pagseguro/ruby/blob/2.5.0-dev/lib/pagseguro/transaction.rb#L129 

Any _dev_ read it, but… :/